tests/queries/0_stateless/01710_minmax_count_projection.sql

```diff
@@ -76,6 +76,6 @@ insert into test select number, number from numbers(1e3);
 
 select count(if(d=4, d, 1)) from test settings force_optimize_projection = 1;
 select count(d/3) from test settings force_optimize_projection = 1;
-select count(if(d=4, Null, 1)) from test settings force_optimize_projection = 1; -- { serverError PROJECTION_NOT_USED }
+select count(if(d=4, Null, 1)) from test settings force_optimize_projection = 1;
```

Comment:
This is wrong. We cannot use implicit projections to count nullable columns. The result looks correct which means `force_optimize_projection` isn't effective in such case. Will investigate.